scripts/close-milestone.sh

```diff
@@ -0,0 +1,11 @@
+set -x -e -o pipefail -u
+
+indented_milestones_json_array=$(gh api -H "Accept: application/vnd.github+json" /repos/VirtusLab/git-machete-intellij-plugin/milestones)
+
+desired_milestone=$( echo $indented_milestones_json_array | jq -c --arg TITLE $1 '.[] | select(.title | contains($TITLE))')
```

Comment:
Don't use vanilla `$1` inside the script like that; name the param first like e.g. in https://github.com/VirtusLab/git-machete-intellij-plugin/blob/master/scripts/git-hooks/machete-status-branch#L9